templates/stackimage/Dockerfile

```diff
@@ -41,15 +41,16 @@ COPY ./LICENSE /licenses/
 USER java_user
 
 ADD ./config /stacks/java-openliberty/config
-ADD ./scripts /stacks/java-openliberty/scripts
 ADD ./priming-app/src /stacks/java-openliberty/priming-app/src
 COPY --chown=1001:0 ./priming-app/pom.xml /stacks/java-openliberty/priming-app/
 
 WORKDIR /stacks/java-openliberty/priming-app
 
-ARG LIBERTY_RUNTIME_VERSION={{.OL_RUNTIME_VERSION}}
+ARG LIBERTY_RUNTIME_VERSION={{.LIBERTY_RUNTIME_VERSION}}
+ARG LIBERTY_RUNTIME_ARTIFACTID={{.LIBERTY_RUNTIME_ARTIFACTID}}
+ARG LIBERTY_RUNTIME_GROUPID={{.LIBERTY_RUNTIME_GROUPID}}
```

Comment:
I wonder too if these three would be simpler as ENV than ARG(s) since you've already got the parameterization provided by the template substitution.   But whatever seems better to you is fine.

Reply:
Agree - Done